src/assets/stylesheets/info-dialog.scss

```diff
@@ -49,6 +62,10 @@
         margin: 10px 0;
         white-space: pre-wrap;
 
+        @media (max-width: 801px) {
```

Comment:
This width was chosen to include mobile and mobile vr browsers, but feels a bit arbitrary.